R/deployDoc.R

```diff
@@ -23,15 +23,16 @@
 #' @family Deployment functions
 #' @export
 deployDoc <- function(doc, ...) {
+  check_installed(
```

Comment:
This is a functional change, since it will prompt interactive users to install the packages rather than immediately stopping; should we announce it in NEWS.md?

Reply:
Ooops, missed this. I'll add momentarily.